main.nf

```diff
@@ -51,6 +48,7 @@ Channel
 ch_splicesites_for_bismark_hisat_align = params.known_splices ? Channel.fromPath(params.known_splices, checkIfExists: true) : Channel.empty()
 
 if( params.aligner =~ /bismark/ ){
+    params.bismark_index = params.genome && params.aligner == 'bismark' ? params.genomes[ params.genome ].bismark ?: false : false
```

Comment:
The relevant change is here

Reply:
So with this, the iGenomes index is only used if `params.aligner == bismark` (which is using `bowtie2`)

Reply:
Ahh sorry, I didn't notice that it had changed, my bad. Great!